src/Microsoft.AspNet.Mvc.Core/Rendering/Html/TemplateBuilder.cs

```diff
@@ -5,9 +5,9 @@
 using Microsoft.AspNet.Mvc.ModelBinding;
 using Microsoft.Framework.Internal;
 
-namespace Microsoft.AspNet.Mvc.Rendering
+namespace Microsoft.AspNet.Mvc.Rendering.Internal
```

Comment:
:ok_hand: then we have some work to do cleaning up the `src/Microsoft.AspNet.Mvc.Core/Internal` folder

in any case @ajaybhargavb the files in the new `Microsoft.AspNet.Mvc.Rendering.Internal` namespace should move to another folder